developers/index.md

```diff
@@ -44,6 +44,8 @@ API documentation for the **latest stable release** is available here:
 <li><a href="https://wpewebkit.org/reference/stable/wpe-web-extension-1.1/">WPEWebExtension 1.1 API documentation</a> (deprecated)</li>
 </ul>
 
+Documentation is also available for a number of [other releases](/reference).
```

Comment:
I would replace other with older.

Reply:
The latest development release is usually _newer_ than the stable one, that's why I used “other” instead of “older” 😓

Reply:
Hm, ok then.